### DISCUSSION TITLE
unsupported apollo for 5.07 firmware version

### DISCUSSION THREAD

**Discussion by player72** [bucanero/apollo-ps4#44] [Q&A] [answered]
can u add 5.07 firmware version for apollo plz :'( @bucanero

**Comment by bucanero** [comment, marked as answer]
See #48 .

I can add firmware detection for 5.07, but keep in mind that you still need goldhen or ps4debug for 5.07
Also, I don't have save-mount patches for 5.07. I could use the patches from 5.05 but I don't know if they'll work or not.

if you have ps4debug or goldhen, I could build a beta version and you can test it out on your 5.07 to see if the patches actually work or not.
